converters/fromZigbee.js

```diff
@@ -738,6 +738,37 @@ const converters = {
             return payload;
         },
     },
+    impulse_metering: {
+        /**
+         * When using this converter also add the following to the configure method of the device:
+         * endpoint.saveClusterAttributeKeyValue('seMetering', {multiplier: 10});
+         */
+        cluster: 'seMetering',
+        type: ['attributeReport', 'readResponse'],
+        options: [
+            exposes.numeric('impulses', exposes.access.SET)
+                .withValueMin(1)
+                .withValueStep(1)
+                .withUnit('Imp./kWh')
+                .withDescription('Configure the impulses per kWh of the power meter. '),
+        ],
+        convert: (model, msg, publish, options, meta) => {
+            if (utils.hasAlreadyProcessedMessage(msg, model)) return;
+            const payload = {};
```

Comment:
Can you use the existing metering converter and apply the impulses afterwards? Use this here: `const payload = fz.metering(model, msg, publish, options, meta)`